Issue #6559: Changed default mail FROM header

Rete2 opened the pull request:
https://github.com/diaspora/diaspora/issues/6559

SuperTux88 committed:
Send public profiles publicly

this will help big profiles, and the informations are public anyway, so no
need to encrypt them.

fixes #3164

Flaburgan committed:
Bump ruby to 2.4

SuperTux88 committed:
Bump diaspora_federation

SuperTux88 committed:
Replace FactoryGirl with Fabricate for federation factories

SuperTux88 committed:
Replace additional_xml_elements with additional_data for relayables

SuperTux88 committed:
New namespace for InvalidRootNode

SuperTux88 committed:
Fix notification factory

SuperTux88 committed:
Hash from federation entity now contains the correct data-type

SuperTux88 committed:
Remove guid and public_key from WebFinger

Related to diaspora/diaspora_federation#39

SuperTux88 committed:
Make Message entity non-relayable

Related to #36

SuperTux88 committed:
Send new MagicEnvelope and EncryptedMagicEnvelope

Related to diaspora/diaspora_federation#30

SuperTux88 committed:
Send only Retractions

Related to diaspora/diaspora_federation#27

SuperTux88 committed:
Remove backward-compatibility for Retractions

SuperTux88 committed:
Send Contact entity for start/stop sharing

Related to diaspora/diaspora_federation#32

SuperTux88 committed:
Send MagicEnvelope as body with correct Content-Type in specs

Related to diaspora/diaspora_federation#30

SuperTux88 committed:
Remove legacy post xml

This was used for the old post fetching

Related to diaspora/diaspora_federation#31

SuperTux88 committed:
Remove old unused salmon method

SuperTux88 committed:
Make Participation entity non-relayable

Related to diaspora/diaspora_federation#35

SuperTux88 committed:
Fix public flag for contact retractions

SuperTux88 committed:
Rename xml_order to signature_order

SuperTux88 committed:
Use build_relayable_federation_entity helper for receive specs

SuperTux88 committed:
Use RFC 7033 webfinger from diaspora_federation gem

SuperTux88 committed:
Send Relayables with parent author when the parent is local

Needed for diaspora/diaspora_federation#64

Commit:
Use ContactRetraction for stop sharing with someone

SuperTux88 committed:
Add tests for ContactRetraction

SuperTux88 committed:
Make Participation entity non-relayable

Flaburgan committed:
Fix height too high on mobile SPV, closes #7402

Flaburgan committed:
Always link comment count text on mobile

Rete2 committed:
Issue #6559: Changed default mail FROM header and tests

SuperTux88 committed:
updated 126 locale files [ci skip]

SuperTux88 committed:
Merge branch 'release/0.6.7.0'

SuperTux88 committed:
Merge branch 'master' into next-minor

SuperTux88 committed:
Merge branch 'next-minor' into develop

SuperTux88 commented:
I think this PR goes to the wrong target repo? You shouldn't change the `develop` branch, so you can start new branches from it.

SuperTux88 committed:
Merge pull request #7480 from Flaburgan/7402-space-comment-mobile-spv

Fix height too high on mobile SPV, closes #7402

SuperTux88 committed:
Merge pull request #7483 from Flaburgan/7393-always-link-comment-mobile

Always link comment count text on mobile

FinnWoelm committed:
Fix: Check for asset precompilation when starting server

When `public/assets/` is a symlink, running `find public/assets ...`
does not return the correct result. By using `public/assets/`, find is
able to return the correct result regardless of whether the folder is an
actual folder or a symlink.

[ci skip]

netagonen commented:
Hi @SuperTux88, we opened a pull request against our origin/develop branch (that means to our own repo) in order to compare our changes. We didn't mean to open a pull request to the upstream/develop just yet :). Our work is still in progress.

SuperTux88 commented:
OK, as long as you don't merge it to your develop, everything is fine 👍 

You can also open an early PR to upstream/develop if you want and add [WIP] to the title (so we know, you are still working on it). That is maybe better/easier to discuss some questions. The only "disadvantage" (can also be an advantage) of opening a PR to upstream is, that it triggers travis runs for each time you push. So you shouldn't push every 5 minutes, but when you push a few times a day, that isn't a problem.

SuperTux88 committed:
Merge pull request #7488 from UpshiftNetwork/fix-assets-check-2

Fix: Check for asset precompilation when starting server

SuperTux88 committed:
Merge pull request #7436 from SuperTux88/federation-update

Federation gem update

SuperTux88 committed:
Update changelog [ci skip]

FinnWoelm committed:
Fix: Use Capybara#fill_in for appending to publisher

Capybara's native#send_key function is slow when it is passed a string
longer than just a few characters. This often results in timeout issues
and Capybara (falsely) reporting feature as failing.

To fix this, we use the faster function #fill_in. This does not trigger
JavaScript events on the input, so we manually trigger them after
fill_in by just sending a single key. This can be any key but since we
do not want to modify the text in the input, non-text keys should be
used. For a list of non-text keys, see
http://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FElement%3Asend_keys

There is an alternative to the above:
1) Use #fill_in to enter all text except for the last character:
    fill_in ..., with: "#{status_message_text} #{txt[0..-2]}"
2) And then use #send_key to send that last character:
    find("#status_message_text").native.send_key(txt.last)

At the moment, both approaches work equally well but the second approach
is documented here just in case it becomes relevant in the future.

SuperTux88 committed:
Merge pull request #7491 from UpshiftNetwork/fix-failing-mention-features

Fix: Use Capybara#fill_in for appending to publisher

FinnWoelm committed:
Fix: Failing 'Signs Up'-Feature Tests

Up to now, scripts were loaded prior to the stylesheet in <head>. When
scripts are loaded before stylesheets, it is possible that scripts
execute on the DOM elements before styles are fully loaded. This can
yield unexpected results.

In the case of the features/desktop/signs_up.feature tests, it would
happen every so often that the scripts ran before styles were loaded.
For example, the app.views.Publisher class (responsible for actions
related to creating a new post) relies on the poll element being either
hidden (or poll options being defined). As per stylesheet, the poll is
hidden by default. But when the scripts run before the stylesheet has
loaded, the poll appears visible to the script and post submission is
disabled.

This is fixed by loading stylesheets prior to scripts. See SO for more
info: https://stackoverflow.com/a/1324720/6451879

SuperTux88 committed:
Merge pull request #7497 from UpshiftNetwork/fix-signs-up-feature-tests

Fix: Failing 'Signs Up'-Feature Tests

netagonen committed:
Replace Diaspora* with pod_name in the FROM header

Add pod name to FROM: header for no sender case

SuperTux88 committed:
Bump rails

SuperTux88 committed:
Bump unicorn

SuperTux88 committed:
Bump responders

SuperTux88 committed:
Bump json

SuperTux88 committed:
Bump devise

SuperTux88 committed:
Bump sidekiq

SuperTux88 committed:
Bump sidekiq-cron

SuperTux88 committed:
Bump uglifier

SuperTux88 committed:
Bump rack-cors

SuperTux88 committed:
Bump autoprefixer-rails

SuperTux88 committed:
Bump compass-rails

Add sprockets as fixed dependency, we don't want to upgrade this by
accident

SuperTux88 committed:
Bump mysql2

SuperTux88 committed:
Bump pg

SuperTux88 committed:
Bump activerecord-import

SuperTux88 committed:
Bump fog-aws

SuperTux88 committed:
Bump carrierwave

SuperTux88 committed:
Bump mini_magick

SuperTux88 committed:
Bump handlebars_assets

SuperTux88 committed:
Bump jquery

SuperTux88 committed:
Bump http_accept_language

SuperTux88 committed:
Bump rails-i18n

SuperTux88 committed:
Bump leaflet-rails

SuperTux88 committed:
Bump nokogiri

SuperTux88 committed:
Bump ruby-oembed

SuperTux88 committed:
Bump twitter-text

SuperTux88 committed:
Bump secure_headers

SuperTux88 committed:
Bump omniauth

SuperTux88 committed:
Bump openid_connect

SuperTux88 committed:
Bump active_model_serializers

SuperTux88 committed:
Bump addressable

SuperTux88 committed:
Bump hamlit

SuperTux88 committed:
Bump will_paginate

SuperTux88 committed:
Bump logging-rails

SuperTux88 committed:
Bump eye

SuperTux88 committed:
Bump asset_sync

SuperTux88 committed:
Bump guard-rubocop

SuperTux88 committed:
Bump rb-fsevent

SuperTux88 committed:
Bump rb-inotify

SuperTux88 committed:
Bump pronto

SuperTux88 committed:
Bump rubocop

SuperTux88 committed:
Bump haml_lint

SuperTux88 committed:
Bump spring

SuperTux88 committed:
Bump fixture_builder

SuperTux88 committed:
Bump capybara

SuperTux88 committed:
Bump database_cleaner

SuperTux88 committed:
Bump poltergeist

SuperTux88 committed:
Bump timecop

SuperTux88 committed:
Bump webmock

SuperTux88 committed:
Bump shoulda-matchers

SuperTux88 committed:
Bump rspec

SuperTux88 committed:
Bump cucumber-rails

SuperTux88 committed:
Bump dependency gems

SuperTux88 committed:
Replace jquery-ui-rails with rails-assets-jquery.ui

SuperTux88 committed:
Bump rails-assets-autosize

SuperTux88 committed:
Bump rails-assets-blueimp-gallery

SuperTux88 committed:
Bump rails-assets-highlightjs

SuperTux88 committed:
Bump rails-assets-markdown-it

SuperTux88 committed:
Use latest ruby-versions

SuperTux88 committed:
Send and receive public flag for profile

skyschub committed:
Merge pull request #7500 from SuperTux88/update-gems

Update gems

SuperTux88 committed:
Install gems for production, to see when compatibility breaks

SuperTux88 committed:
Cache assets-cache for cucumber on travis

SuperTux88 committed:
Merge pull request #7501 from SuperTux88/send-public-profile

Send public profiles publicly

SuperTux88 committed:
Merge pull request #7259 from Flaburgan/bump-ruby-2.4

Bump ruby to 2.4

SuperTux88 committed:
Merge pull request #7502 from SuperTux88/cucumber-first-login-timeout

Cache assets-cache for cucumber on travis

SuperTux88 committed:
Exclude ignored people from followed tags when building stream-relation

This improves the main stream when you have less or inactive contacts
and ignored an active person that posts many posts with tags. When the
person is only removed after collecting all post ids, you don't see
posts with tags from other people.

SuperTux88 committed:
Merge pull request #7503 from SuperTux88/quickfix-multi-stream-ignore-users-with-tags

Exclude ignored people from followed tags when building stream-relation
